### PR TITLE
[13.0] [MIG] stock_picking_product_assortment: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT27475

please @pedrobaeza @victoralmau review this

### CODE REVIEW

File: stock_picking_product_assortment/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    )
+
+    @api.depends("partner_id")
+    def _compute_product_assortment_ids(self):
+        # If we don't initialize the fields we get an error with NewId
+        self.whitelist_product_ids = self.env["product.product"]
+        self.blacklist_product_ids = self.env["product.product"]
+        self.has_whitelist = False
+        self.has_blacklist = False
+        if self.partner_id and self.picking_type_id.code == "outgoing":
```
Comment (victoralmau) [inline stock_picking_product_assortment/models/stock_picking.py:33]: Limit to outgoing is really necessary?

**Review by victoralmau — APPROVED** [review]

File: stock_picking_product_assortment/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
```
Comment (Tardo) [inline stock_picking_product_assortment/models/stock_move.py:8]: I don't understand what you intend to do here

File: stock_picking_product_assortment/models/stock_move_line.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
+
+
+class StockMoveLine(models.Model):
```
Comment (Tardo) [inline stock_picking_product_assortment/models/stock_move_line.py:8]: I don't understand what you intend to do here

**Comment by CarlosRoca13** [comment]
@Tardo sorry, don't remember to set the commets explaining the purpose, I add them yet

**Comment by CarlosRoca13** [comment]
@Tardo can you review it again?

**Comment by sergio-teruel** [comment]
Plase increase the coverge

**Comment by CarlosRoca13** [comment]
@sergio-teruel done!

**Review by sergio-teruel — APPROVED** [review]

File: stock_picking_product_assortment/models/stock_move.py (hunk 2 above)
Comment (sergio-teruel) [reply stock_picking_product_assortment/models/stock_move.py:8]: Is due to a dynamic field domains

**Pull request merged**